From ddb13d29c6289c8516da19e16e8fa012956cc849 Mon Sep 17 00:00:00 2001
From: Nanda Krishnan <quic_nandkris@quicinc.com>
Date: Thu, 13 Oct 2022 11:54:20 +0530
Subject: [PATCH 249/500] [linux] Avoid invalidate if recycled_for_ds flag is
 set

Avoiding invalidation in wifi if recycled_for_ds flag is set

Change-Id: I1841d821597a27833437c203c0adc2dc4ba0001a
Signed-off-by: Nanda Krishnan <quic_nandkris@quicinc.com>
---
 include/linux/skbuff.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index de5d19c7318b..20ba9efb984e 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -993,6 +993,8 @@ struct sk_buff {
 	__u8			fast_xmit:1;
 	__u8			fast_forwarded:1;
 	/* 1 or 3 bit hole */
+	/* Flag for recycle in PPE DS */
+	__u8			recycled_for_ds:1;
 
 #if defined(CONFIG_NET_SCHED) || defined(CONFIG_NET_XGRESS)
 	__u16			tc_index;	/* traffic control index */
-- 
2.34.1

